manifest.xml

```diff
@@ -9,5 +9,4 @@
   <logo>http://</logo>
   <depend package="base/cmake" />
   <depend package="base/types" />
-  <depend package="drivers/hidapi" />
```

Comment:
Done.

*However*, I've purged the libhidapi-dev, removed the package and call `aup` in the rock root folder and the OS dependency still is not installing automatically and so the `amake` is breaking.